core/pkg/resultshandling/printer/v2/sarifprinter.go

```diff
@@ -110,57 +111,81 @@ func (sp *SARIFPrinter) addResult(scanRun *sarif.Run, ctl reportsummary.IControl
 		})
 }
 
-func (sp *SARIFPrinter) PrintImageScan(context.Context, *models.PresenterConfig) {
+func (sp *SARIFPrinter) PrintImageScan(scanResults *models.PresenterConfig) error {
+	if scanResults == nil {
+		return fmt.Errorf("no no image vulnerability data provided")
+	}
+
+	presenterConfig, err := presenter.ValidatedConfig("sarif", "", false)
```

Comment:
TODO: We should have const for printer names:
```suggestion
	presenterConfig, err := presenter.ValidatedConfig(SarifFormat, "", false)
```